C/Cairo/Compat.toml

```diff
@@ -57,3 +57,9 @@ julia = "0.6-0.7"
 
 ["0.5.3-0.5.6"]
 julia = "0.6-1.3"
+
+["0.6"]
+BinDeps = ["0.3.21-*"]
```

Comment:
https://github.com/JuliaRegistries/General/pull/111#discussion_r277135089

Reply:
Also, consider using semver bounds instead of allowing all future releases of packages and julia.

Reply:
I am not sure what I should do? What do you want me to change?

Reply:
The first comment was just a reason why I did not merge this right now. The second one was a suggestion to use
```toml
[compat]
BinDeps = "0.3.21"
Compat = "0.52"
Graphics = "0.1"
julia = "≥ 0.7.0"
```
instead of 
```toml
[compat]
BinDeps = "≥ 0.3.21"
Compat = "≥ 0.52.0"
Graphics = "≥ 0.1.0"
julia = "≥ 0.7.0"
```
because in the latter you claim compatibility with all future releases of these packages, which obviously isn't true.

Reply:
I still do not get the first comment. Is `["0.3.21-*"]` wrong? I used the converter script from Pkg.

Reply:
Yea its not your project file that is wrong, its Registrator.

Reply:
Ah thanks, so anything I can do? I could change to the stricter version and then retrigger Registrator (using the same command?)?